doc-checking-app-api/src/test/java/uk/gov/di/authentication/app/lambda/DocAppCallbackHandlerTest.java

```diff
@@ -112,7 +113,8 @@ class DocAppCallbackHandlerTest {
     private final Session session = new Session(SESSION_ID).setEmailAddress(TEST_EMAIL_ADDRESS);
 
     private final ClientSession clientSession =
-            new ClientSession(generateAuthRequest().toParameters(), null, null, null);
+            new ClientSession(
+                    generateAuthRequest().toParameters(), null, (VectorOfTrust) null, null);
```

Comment:
It might be a good idea to move the tests to the vector of trust list approach as well?

Reply:
I started doing that but it got quite messy, was planning to do it in a separate PR

Reply:
okay sure 👍